scribejava-apis/src/main/java/com/github/scribejava/apis/QQApi.java

```diff
@@ -0,0 +1,93 @@
+/*
+  Copyright 2012 - 2014 Zhang Zhenli
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
```

Comment:
Do you really need this Copyright?
We have one here
https://github.com/scribejava/scribejava/blob/master/LICENSE.txt

And why 2012 - 2014?